### DISCUSSION TITLE
discover uploads

### DISCUSSION THREAD

**Discussion by emahuni** [directus/directus#18740] [Feature Requests]
### Summary

So there is this situation where you need to use all uploads in the already existing uploads folder that otherwise don't appear in the file library due to the fact that they don't exist in the DB, without manually uploading them. They were uploaded by another framework or previous installation of Directus. I just wished directus had a button in file library to discover uploads and enter them into the DB.

### Basic Example

I tried Directus using SQLite, then decided to redo it using Postgres. There were a lot of uploads that were done and wanted to see how to quickly recreate with a fresh DB, but wanted the uploads metadata. 

### Motivation

This will make it easier to import files that were uploaded by other systems that one will be migrating from into Directus.  ie: another CMS that has uploads already but obviously has an incompatible uploads table. The simple thing is just to make all these uploads available in the files lib.

### Detailed Design

Just an extra button in the File Library.

The logic that will handle it would use the same upload logic on each file that is discovered from the uploads folder.

### Requirements List

none

### Drawbacks

the user will have to do some manual work associating the uploads with their respective collections.

### Alternatives

move the files to another place and then upload them manually

### Adoption Strategy

they can just use it when necessary, it has no breaking behaviour

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
